src/transport/SessionManager.cpp

```diff
@@ -532,7 +532,7 @@ void SessionManager::SecureUnicastMessageDispatch(const PacketHeader & packetHea
 
     Transport::SecureSession * secureSession = session.Value()->AsSecureSession();
 
-    if (!secureSession->IsDefunct() && !secureSession->IsActiveSession())
+    if (!secureSession->IsDefunct() && !secureSession->IsActiveSession() && !secureSession->IsInactive())
```

Comment:
Perhaps a comment should be added here explaining why we're letting inactive sessions to still be processed, because to a layman reader, doing this doesn't make a ton of sense no?